src/main/java/uk/gov/register/resources/IndexSizePagination.java

```diff
@@ -91,10 +91,14 @@ public String getNextPageLink() {
     public String getPreviousPageLink() {
         return String.format("?" + INDEX_PARAM + "=%s&" + SIZE_PARAM + "=%s", getPreviousPageNumber(), pageSize());
     }
-
+    
     public int pageSize() {
         return pageSize;
     }
+    
+    public String getFormattedPageQueryParams() {
+        return String.format("?%s=%s&%s=%s", INDEX_PARAM, pageIndex, SIZE_PARAM, pageSize);
```

Comment:
Is there a Java library for generating query strings rather than doing it by hand?

Reply:
Not sure, although we do use `String.format` in other places in the codebase.

Reply:
Looking at this more, there are things out there such as `UriBuilder` to build an entire URI, though I'm not sure how applicable this is for the current situation, given that these URIs are all generated within the template files.